docs/reference-guide/modules/queries/pages/implementations.adoc

```diff
@@ -141,4 +164,94 @@ public class AxonConfig {
 }
 ----
 --
-====
\ No newline at end of file
+====
+
+== Subscription query infrastructure
+
+Subscription queries allow clients to receive an initial result and then continue receiving updates as long as the subscription is active. The subscription query infrastructure consists of several components working together.
+
+=== `QueryUpdateEmitter`
+
+The `QueryUpdateEmitter` is responsible for emitting updates to active subscription queries. The `QueryUpdateEmitter` is context-aware and must be created from the `ProcessingContext`:
+
+[source,java]
+----
+import org.axonframework.messaging.queryhandling.QueryUpdateEmitter;
+import org.axonframework.messaging.eventhandling.EventHandler;
+
+@Component
+public class CardSummaryProjection {
+
+    @EventHandler
+    public void on(CardRedeemedEvent event, ProcessingContext context) {
+        // Create a context-aware emitter
+        QueryUpdateEmitter emitter = QueryUpdateEmitter.forContext(context);
+
+        // Update the model
+        CardSummary summary = updateModel(event);
+
+        // Emit update to subscription queries
+        emitter.emit(FetchCardSummaryQuery.class,
+                    query -> query.getCardSummaryId().equals(event.getCardId()),
+                    summary);
+    }
+}
+----
+
+The emitter filters subscription queries based on the provided predicate and emits the update only to matching subscriptions. This allows fine-grained control over which subscribers receive which updates.
```

Comment:
Would be good to note Axon Framework does this for the user **if** they are dealing with any annotated methods. A parameter resolver takes care of it, essentially. Hence, constructing the emitter like this is only mandatory when manually registering a query handler.